src/sage/misc/sageinspect.py

```diff
@@ -370,7 +357,7 @@ def _extract_embedded_signature(docstring, name):
 
 class BlockFinder:
     """
-    Provide a tokeneater() method to detect the end of a code block.
+    Provide a :meth:`tokeneater` method to detect the end of a code block.
 
     This is the Python library's :class:`inspect.BlockFinder` modified
     to recognize Cython definitions.
```

Comment:
This `:meth:` role is new and the reason why this didn't fail before and fails now. Note that the `:class:` role is also not translated, however it's past the first 50 characters that are tested.